src/wrap-require.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from 'path'
+
+export const wrapRequire = new Proxy(__non_webpack_require__, {
+  apply: (target, thisArg, [moduleID]) => {
+    if (moduleID.startsWith('.')) {
+      moduleID = path.join(process.cwd(), moduleID)
```

Comment:
Should we use `path.resolve` here instead?
```suggestion
      moduleID = path.resolve(moduleID)
```